payjoin-ffi/src/test_utils.rs

```diff
@@ -191,7 +203,7 @@ pub fn init_bitcoind_sender_receiver() -> Result<Arc<BitcoindEnv>, FfiError> {
 }
 
 #[uniffi::export]
-pub fn example_url() -> Url { EXAMPLE_URL.clone().into() }
+pub fn example_url() -> String { EXAMPLE_URL.to_string() }
```

Comment:
Where is this used and why would a `payjoin_ffi::Url` return type with `payjoin_ffi::Url::as_str` conversion NOT be more appropriate at callsites that require a string over Url?